index.html

```diff
@@ -60,8 +60,106 @@
+      <div class="col s4">
+        <table>
+          <thead>
+            <tr>
+              <td>Party</td>
+              <td>Voter Percentage</td>
+              <td>Congress Percentage</td>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>Democrat</td>
+              <td id='voter-affiliation-democrat'></td>
+              <td id='voter-representation-democrat'></td>
+            </tr>
+            <tr>
+              <td>Republican</td>
+              <td id='voter-affiliation-republican'></td>
+              <td id='voter-representation-republican'></td>
+            </tr>
+            <tr>
+              <td>Libertarian</td>
+              <td id='voter-affiliation-libertarian'></td>
+              <td id='voter-representation-libertarian'></td>
+            </tr>
+            <tr>
+              <td>Unaffiliated</td>
+              <td id='voter-affiliation-unaffiliated'></td>
+              <td id='voter-representation-unaffiliated'></td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+
+      <div class="col s4">
+        <table>
+          <thead>
+            <tr>
+              <td>Race</td>
+              <td>Voter Percentage</td>
+              <td>Congress Percentage</td>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>White</td>
+              <td id='voter-affiliation-white'></td>
+              <td id='voter-representation-white'></td>
+            </tr>
+            <tr>
+              <td>Black</td>
+              <td id='voter-affiliation-black'></td>
+              <td id='voter-representation-black'></td>
+            </tr>
+            <tr>
+              <td>American Indian</td>
+              <td id='voter-affiliation-americanindian'></td>
+              <td id='voter-representation-americanindian'></td>
+            </tr>
+            <tr>
+              <td>Hispanic</td>
+              <td id='voter-affiliation-hispanic'></td>
+              <td id='voter-representation-hispanic'></td>
+            </tr>
+            <tr>
+              <td>Other</td>
+              <td id='voter-affiliation-other'></td>
+              <td id='voter-representation-other'></td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
 
+      <div class="col s4">
+        <table>
+          <thead>
+            <tr>
+              <td>Sex</td>
+              <td>Voter Percentage</td>
+              <td>Congress Percentage</td>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>Male</td>
+              <td id='voter-affiliation-male'></td>
+              <td id='voter-representation-male'></td>
+            </tr>
+            <tr>
+              <td>Female</td>
+              <td id='voter-affiliation-female'></td>
+              <td id='voter-representation-female'></td>
+            </tr>
+          </tbody>
+        </table>
+      <div>
```

Comment:
This should be a closing `</div>` and it is also missing the closing `</div>` for the row.